### PR TITLE
DOT-3204 StoreAdmin > API > CRUD shipping zones

### PR DESCRIPTION
**Pull request by mikhalkov5**
https://nextmediama.atlassian.net/browse/DOT-3204

### CODE REVIEW

File: store-admin/shipping-zones/create.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+| `name`         | string     | shipping zone name          | yes       |
+| `display_name` | string     | shipping zone display name  | yes       |
+| `countries`    | array,null | shipping zone countries     | no        |
+| `is_free`      | boolean    | is free shipping            | no        |
+| `is_active`    | boolean    | is shipping zone active    | no         |
+| `rate_type`    | string     | (prince, weight)            | no        |
+| `rates` | [ShippingRate](#ShippingRate) | shipping zone rates  | no   |
+
+## ShippingRate
+
```
Comment (5baddi) [inline store-admin/shipping-zones/create.md:22]: this belong to parameters i guess we can merge it in one table

File: store-admin/shipping-zones/delete.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Delete a Shipping Zone
+
+Endpoint: `https://api.youcan.shop/shipping-zones/{id}`
+
+Method: `DELETE`
+
+<a name="response"></a>
+
+## Responses
```
Comment (5baddi) [inline store-admin/shipping-zones/delete.md:9]: We have only one response instead of responses

File: store-admin/shipping-zones/get.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Get a Shipping Zone
+
+Endpoint: `https://api.youcan.shop/shipping-zones/{id}`
+
+Method: `GET`
+
+## Response
```
Comment (5baddi) [inline store-admin/shipping-zones/get.md:7]: Missing not found response

File: store-admin/shipping-zones/rates/create.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        },
+        {
+            "id": "c8af56ee-1c76-4c31-85ff-08e865b8a4ae",
+            "min": 30.1,
+            "max": null,
+            "price": 30.1
+        }
+    ]
+}
+```
```
Comment (5baddi) [inline store-admin/shipping-zones/rates/create.md:63]: Missing not found response

File: store-admin/shipping-zones/rates/update.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+```json
+{
+    "message": "Shipping rate updated successfully",
+    "http_code": 200,
+    "code": "UPDATED",
+    "type": "success",
+    "data": null
+}
+```
```
Comment (5baddi) [inline store-admin/shipping-zones/rates/update.md:34]: Missing not found response

**Review by 5baddi — COMMENTED** [review]
Left some comments

File: store-admin/shipping-zones/create.md (hunk 1 above)
Comment (mikhalkov5) [reply store-admin/shipping-zones/create.md]: Fixed 🙏

File: store-admin/shipping-zones/rates/create.md (hunk 4 above)
Comment (mikhalkov5) [reply store-admin/shipping-zones/rates/create.md:63]: fixed

File: store-admin/shipping-zones/rates/update.md (hunk 5 above)
Comment (mikhalkov5) [reply store-admin/shipping-zones/rates/update.md:34]: fixed

File: store-admin/shipping-zones/delete.md (hunk 2 above)
Comment (mikhalkov5) [reply store-admin/shipping-zones/delete.md]: fixed

**Pull request merged**